response_writer_test.go

```diff
@@ -185,4 +185,11 @@ func Test_ResponseWriter(t *testing.T) {
 		So(resp.Code, ShouldEqual, http.StatusOK)
 		So(resp.Body.String(), ShouldEqual, "data: Hello\n\ndata: Hello\n\n")
 	})
+
+	Convey("Response writer with http/2 push", t, func() {
+		resp := httptest.NewRecorder()
+		rw := NewResponseWriter("GET", resp)
+		_, ok := rw.(http.Pusher)
+		So(ok, ShouldBeTrue)
```

Comment:
No test case for the call of `pusher.Push` method?

Reply:
I will not lie to you. As I never used Convoy I got a little bit confused how to use it in this more complex use case. As I have other demands from my employer if you could help me out on this or give a waiver in this method I would appreciate it

Reply:
No worries! I'll see what I can do and push to this PR.